microsoft-edge/webview2/how-to/debug-visual-studio-code.md

```diff
@@ -11,16 +11,19 @@ ms.date: 02/11/2022
 # Debug WebView2 apps with Visual Studio Code
 
 Use Microsoft Visual Studio Code to debug scripts that run in WebView2 controls.  <!-- Make sure you're using Visual Studio Code version [insert build here] or later. -->
+Visual Studio Code has a built-in [debugger](https://code.visualstudio.com/docs/nodejs/browser-debugging) for browser debugging.
```

Comment:
```suggestion
Visual Studio Code has a built-in debugger for browser debugging.  See [Browser debugging in VS Code](https://code.visualstudio.com/docs/nodejs/browser-debugging).
```
per repo style guide, d/k if shown in wiki > Edge Dev Style Guide.  
And per corp-level (tho case-by-case) style guide, show actual destination heading in link text.  (in most cases, where fits)